src/applications/gi/components/profile/EstimateYourBenefitsForm.jsx

```diff
@@ -391,67 +400,62 @@ class EstimateYourBenefitsForm extends React.Component {
           onChange={this.handleInputChange}
           onFocus={this.handleEYBInputFocus}
         />
-        {this.props.inputs.yellowRibbonRecipient === 'yes' ? (
```

Comment:
moved checks like this for one parent to one conditionally displayed question to the [ExpandingGroup](https://github.com/usds/us-forms-system/blob/master/src/js/components/ExpandingGroup.jsx) element property `open` as [ExpandingGroup](https://github.com/usds/us-forms-system/blob/master/src/js/components/ExpandingGroup.jsx) will return null if `open` is false